src/pages/workspace/accounting/PolicyAccountingPage.tsx

```diff
@@ -18,72 +19,43 @@ import useTheme from '@hooks/useTheme';
 import useThemeStyles from '@hooks/useThemeStyles';
 // import useWaitForNavigation from '@hooks/useWaitForNavigation';
 import useWindowDimensions from '@hooks/useWindowDimensions';
+import {removePolicyConnection} from '@libs/actions/connections';
 import Navigation from '@navigation/Navigation';
 import AdminPolicyAccessOrNotFoundWrapper from '@pages/workspace/AdminPolicyAccessOrNotFoundWrapper';
 import FeatureEnabledAccessOrNotFoundWrapper from '@pages/workspace/FeatureEnabledAccessOrNotFoundWrapper';
 import PaidPolicyAccessOrNotFoundWrapper from '@pages/workspace/PaidPolicyAccessOrNotFoundWrapper';
-import withPolicy from '@pages/workspace/withPolicy';
-import type {WithPolicyProps} from '@pages/workspace/withPolicy';
+import type {WithPolicyAndFullscreenLoadingProps} from '@pages/workspace/withPolicyAndFullscreenLoading';
+import withPolicyAndFullscreenLoading from '@pages/workspace/withPolicyAndFullscreenLoading';
 import type {AnchorPosition} from '@styles/index';
 import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
 import ROUTES from '@src/ROUTES';
+import type {Policy, PolicyConnectionSyncProgress} from '@src/types/onyx';
 
-function WorkspaceAccountingPage({policy}: WithPolicyProps) {
+type PolicyAccountingPageOnyxProps = {
+    /** From Onyx */
+    /** Bank account attached to free plan */
+    connectionSyncProgress: OnyxEntry<PolicyConnectionSyncProgress>;
+};
+
+type PolicyAccountingPageProps = WithPolicyAndFullscreenLoadingProps &
+    PolicyAccountingPageOnyxProps & {
+        /** Policy values needed in the component */
+        policy: Policy;
+    };
```

Comment:
```suggestion
type PolicyAccountingPageProps = WithPolicyAndFullscreenLoadingProps & PolicyAccountingPageOnyxProps;
```

`policy` type is included in `WithPolicyAndFullscreenLoadingProps`

Reply:
I'm overriding `policy` because it is possibly `null` thanks to `OnyxEntry`, and we don't want to render this page if the policy is missing. Since we are wrapping this page with `withPolicyAndFullscreenLoading`, this will never happen.

Having the `policy` possibly `null` is incorrect, it will never be, and forces me to add useless checks.

Reply:
>Since we are wrapping this page with withPolicyAndFullscreenLoading, this will never happen.

I agree with this observation. That is the case for all places where we use `withPolicyAndFullscreenLoading` right? In that case, I think this is a mistake in `WithPolicyAndFullscreenLoadingProps`, which should be fixed separately from this PR?